docs/Developers/Newcomers_Guide.rst

```diff
@@ -393,7 +393,19 @@ Step 4. Sending Your Changes
    is no asterisk next to the master branch.
 
 Now that you've fixed the issue, you've tested it, and you think it is ready
-to be merged, create a commit and push it to your fork, using:
+to be merged, add your updated file to the staging area:
+
+::
+
+    $ git add <file/folder_name>
+
+Create a commit using the following command:
+
+::
+
+    $ git commit
+
```

Comment:
here add that this will open a text editor to write commit message.

Reply:
Done